.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Please do not ask general questions here! There are [comunnity
+contact](https://github.com/qtile/qtile#community) options for that.
```

Comment:
I'm +1 for the template but I really don't mind the questions here. I actually prefer here than on the mailing list, for example.

Reply:
If properly tagged, questions on the bugtracker are great to feed the wiki :slightly_smiling_face:

Reply:
I can drop this, but then it seems like we should dismantle the mailing list all together if nobody wants to use it.

Reply:
I like this idea, but then to each his favorites tools I guess, so I wouldn't mind keeping the current status to reach more people. I didn't find this unmanageable so far. The only thing I don't want is adding more tools than we have currently, e.g. dealing with another chat than IRC unless we all stop using IRC.

Reply:
Ok. I find it annoying that we use issues for people to ask questions. So maybe we can all agree that we can close questions immediately, or something? I'd prefer for the mailing list to be about qustions, issues to be about issues, etc.